### DISCUSSION TITLE
File indoor_ds_new.ckpt file? [Closed]

### DISCUSSION THREAD

**Discussion by Buzzolan** [zju3dv/LoFTR#160] [Q&A]
Hi,
I solved the problem. 
Thank you

**Comment by Buzzolan** [comment]
Hi, you can find the file at this link
<https://drive.google.com/drive/folders/1DOcOPZb3-5cWxLqn256AhwUVjBPifhuf>
inside "weights" folder.

Il giorno mer 16 mar 2022 alle ore 11:45 yifei-ding <
***@***.***> ha scritto:

**Reply by algr2397** [reply]
in which folder i have to put the weights?

**Reply by MonCheri99** [reply]
Inside "weights" folder. You have to download it from the link above.
